test/docker/Dockerfile.testhost

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:focal
```

Comment:
Are these expected to be run on another host and pushed to a repository somewhere, or are these Dockerfiles run everytime?

Reply:
They are run every time, but because of the Docker cache it should only do anything when there is an update to the base image. For example, in [this run](https://github.com/AWSjswinney/arm64-python-wheel-tester/runs/3119350264?check_suite_focus=true) execution of that step only took 1 second.